model/clusters_mgmt/v1/htpasswd_users_resouce.model

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright (c) 2019 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages the collection of users in an _HTPasswd_ IDP of a cluster.
+resource HtpasswdUsers {
```

Comment:
Would be nice to have this name consistent with the name of the type: `HTPasswdUsers`.

Reply:
This causes some issues because of the camel case to snake case transformation. HTPasswdUsers is mapped to ht_passwd_users which is not the endpoint's name...

Reply:
The URL segment is determined by the name of the locator, not by the name of the resource.